src/openeo_aggregator/backend.py

```diff
@@ -6,9 +6,11 @@
 from collections import defaultdict
 from typing import List, Dict, Union, Tuple, Optional, Iterable, Iterator, Callable, Any
 
+import dateutil
 import flask
 
 import openeo_driver.util.view_helpers
+from flask import url_for, has_app_context
```

Comment:
I'd avoid importing these in backend.py like this and just use `flask.url_for`to be more explicit about the origin of these functions